content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -94,21 +94,25 @@ The following ports and protocols are used by the Istio sidecar proxy (Envoy).
 |----|----|----|----|
 | 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
 | 15001 | TCP | Envoy outbound | No |
+| 15004 | HTTP | Debug port | Yes |
 | 15006 | TCP | Envoy inbound | No |
-| 15008 | TCP | Envoy tunnel port (inbound) | No |
+| 15008 | H2 | HBONE mTLS tunnel port | No |
+| 15009 | H2C | HBONE port for secure networks | No |
```

Comment:
This is not ready, right? Why are we publishing it already.

Reply:
15008 was already published - I just updated the doc.

And we are starting to use them ( via bootstrap or envoy filter ) - should be reserved and documented.